homework.js

```diff
@@ -0,0 +1,41 @@
+// #1
+
+const summ = function (n) {
+	let vr = 0;
+  for (let i=0; i <= n.length; i++) {
+	  vr += i; 
+}
+return vr 
+};
+
+// #2
+const summ = function (n) {
+	let vr = 0;
+  for (let i=0; i < n.length; i++) {
+    if (n[i] % 2 === 0){
+      vr += i;   
+    } 
+}
+return vr 
+};
+
+// #3
+
+const summ = function (n) {
+	let vr = 0;
+  let div = 0;
+  for (let i=0; i <= n.length; i++) {
+    vr += i;
+    
+    div += vr /n.length
+}
+return div 
+};
+
+
+//#4
+
+const rev = function(v) {
+  v = v.reverse()
```

Comment:
you can't use the function `reverse()` on strings as it only works with arrays.
The trick to convert the string to array using `split("")`, using the `reverse()` function and then convert it back to string using `join()`